addons/mail/static/src/core/web/store_service_patch.js

```diff
@@ -183,3 +184,7 @@ const StorePatch = {
     onLinkFollowed(fromThread) {},
 };
 patch(Store.prototype, StorePatch);
+
+registry.category("actions").add("mail.store_insert", function storeInsertAction(env, action) {
+    env.services["mail.store"].insert(action.params);
+});
```

Comment:
```suggestion
registry
    .category("actions")
    .add("mail.store_insert", (env, action) => env.services["mail.store"].insert(action.params));
```

why do we need a named function? :pick: